docs/modules/ROOT/pages/navigating.adoc

```diff
@@ -89,6 +90,12 @@ Simply click the *Account* > *Add New* button on any account tab to add a new ac
 The new account will appear as a new tab in the settings dialog, where you can adjust its settings at any time.
 Use *Account* > *Remove* to delete accounts.
 
+[NOTE]
+====
+ownCloud server must have {oauth2-app-url}[the OAuth2 app] installed, configured, and enabled to use Two-Factor Authentication.
```

Comment:
```suggestion
To use *Two-Factor Authentication* (2FA), ownCloud server must have {oauth2-app-url}[the OAuth2 app] installed, configured, and enabled.
```

I'd try to make "Two-Factor Authentication" more prominent.

Reply:
That's an excellent suggestion.